sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py

```diff
@@ -0,0 +1,251 @@
+            country_code,
+            search_request,
+            **kwargs
+        )
+    
+    @distributed_trace
+    def begin_update_phone_number_capabilities(
+            self, 
+            phone_number, 
+            sms, 
+            calling, 
+            **kwargs
+    ):
+        # type: (...) -> LROPoller["_models.AcquiredPhoneNumber"]
+        """Updates the capabilities of a phone number.
+
+        :param phone_number: The phone number id in E.164 format. The leading plus can be either + or
+            encoded as %2B, e.g. +11234567890.
+        :type phone_number: str
+        :param calling: Capability value for calling.
+        :type calling: str or ~azure.communication.phonenumbers.models.PhoneNumberCapabilityValue
+        :param sms: Capability value for SMS.
+        :type sms: str or ~azure.communication.phonenumbers.models.PhoneNumberCapabilityValue
+        :keyword callable cls: A custom type or function that will be passed the direct response
+        :keyword str continuation_token: A continuation token to restart a poller from a saved state.
+        :keyword polling: Pass in True if you'd like the LROBasePolling polling method,
+            False for no polling, or your own initialized polling object for a personal polling strategy.
+        :paramtype polling: bool or ~azure.core.polling.PollingMethod
+        :keyword int polling_interval: Default waiting time between two polls for LRO operations if no Retry-After header is present.
+        :rtype: ~azure.core.polling.LROPoller[AcquiredPhoneNumber]
+        """
+        return self._phone_number_client.phone_numbers.begin_update_capabilities(
+            phone_number,
+            calling,
+            sms,
+            **kwargs
+        ) 
+
+    @distributed_trace
+    def get_phone_number(
+            self, 
+            phone_number,
+            **kwargs
+    ):
+        # type: (...) -> AcquiredPhoneNumber
+        """Gets the details of the given acquired phone number.
+
+        :param phone_number: The acquired phone number whose details are to be fetched in E.164 format,
+         e.g. +11234567890.
+        :type phone_number: str
+        :keyword callable cls: A custom type or function that will be passed the direct response
+        :rtype: ~azure.communication.phonenumbers.models.AcquiredPhoneNumber
+        """
+        return self._phone_number_client.phone_numbers.get_by_number(
+            phone_number,
+            **kwargs
+        )
+
+    @distributed_trace
+    def list_acquired_phone_numbers(
+        self, 
+        **kwargs
+    ):
+         # type: (...) -> ItemPaged[AcquiredPhoneNumbers]
+        """Gets the list of all acquired phone numbers.
+
+        :param skip: An optional parameter for how many entries to skip, for pagination purposes. The
+         default value is 0.
+        :type skip: int
+        :param top: An optional parameter for how many entries to return, for pagination purposes. The
```

Comment:
We don't tend to use the `top` parameter - is it's impact is misleading.
For example - I assume this is used to control the number of results per page, rather than the total number of results returned?
If so, use the name `results_per_page`.